rs/bitcoin/consensus/src/payload_builder.rs

```diff
@@ -223,9 +224,11 @@ impl BitcoinPayloadBuilder {
             // maximum block size of the IC is also 4MiB. This makes it impossible to transport a
             // BTC block via an IC block, since including the metadata would make the block too
             // large. We therefore allow a response to be oversized, if it is the first response in
-            // the block. Since we tolerate up to 2x the size margin currently, this will pass validation
+            // the block AND the payload builder has the highest prioroty.
```

Comment:
I think we should either write 
```suggestion
            // the block.
```
or
```suggestion
            // the payload AND the payload builder has the highest priority.
```